### PR TITLE
jfrog-cli-go 1.3.2

### PR DESCRIPTION
**Pull request by romangurevitch**
- [ ] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document? yes
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change? yes
- [ ] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)? yes
- [ ] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)? yes

---

### CODE REVIEW

File: Formula/jfrog-cli-go.rb [hunk 1]
```diff
@@ -30,6 +30,6 @@ def install
   end
 
   test do
-    assert_equal "jfrog version 1.3.1", shell_output("#{bin}/jfrog -v").chomp
+    assert_equal "jfrog version 1.3.2", shell_output("#{bin}/jfrog -v").chomp
```
Comment (MikeMcQuaid) [inline Formula/jfrog-cli-go.rb]: `assert_equal "jfrog version #{version}"` will save this being updated every time.

**Comment by romangurevitch** [comment]
@MikeMcQuaid, we have updated the patch according to your recommendation.

File: Formula/jfrog-cli-go.rb [hunk 2]
```diff
@@ -3,8 +3,8 @@
 class JfrogCliGo < Formula
   desc "command-line interface for Jfrog Artifactory and Bintray"
   homepage "https://github.com/JFrogDev/jfrog-cli-go"
-  url "https://github.com/JFrogDev/jfrog-cli-go/archive/1.3.1.tar.gz"
-  sha256 "4684813c256e8124623fdcb5089606cc425862fce7dbb8ed4f7e18b1debd297e"
+  url "https://github.com/JFrogDev/jfrog-cli-go/archive/#{version}.tar.gz"
```
Comment (MikeMcQuaid) [inline Formula/jfrog-cli-go.rb]: Sorry, leave the 1.3.1 in here.

Comment (romangurevitch) [inline Formula/jfrog-cli-go.rb]: No problem, done.

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew and patience! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**